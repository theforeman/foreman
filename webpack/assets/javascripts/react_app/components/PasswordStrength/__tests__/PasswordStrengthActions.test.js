import {
  updatePassword,
  updatePasswordConfirmation,
} from '../PasswordStrengthActions';

describe('PasswordStrength actions', () => {
  it('should update password', () =>
    expect(updatePassword('some-password')).toMatchSnapshot());

  it('should update password-confirmation', () =>
    expect(updatePasswordConfirmation('some-password')).toMatchSnapshot());
});
