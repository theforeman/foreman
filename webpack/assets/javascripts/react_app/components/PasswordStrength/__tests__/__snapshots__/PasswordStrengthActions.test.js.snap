// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PasswordStrength actions should update password 1`] = `
Object {
  "payload": "some-password",
  "type": "PASSWORD_STRENGTH_PASSWORD_CHANGED",
}
`;

exports[`PasswordStrength actions should update password-confirmation 1`] = `
Object {
  "payload": "some-password",
  "type": "PASSWORD_STRENGTH_PASSWORD_CONFIRMATION_CHANGED",
}
`;
